cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -28,26 +28,280 @@ Cask Data Application Platform Release Notes
+  password was empty.
+
+- `CDAP-3854 <https://issues.cask.co/browse/CDAP-3854>`__ - Added the capability to
+  configure Kafka topic for logs and notifications using the ``cdap-site.xml``.
+
+- `CDAP-4060 <https://issues.cask.co/browse/CDAP-4060>`__ - Added the status for custom
+  actions in workflow diagrams.
+
+- `CDAP-4143 <https://issues.cask.co/browse/CDAP-4143>`__ - Fixed a problem with the
+  database source where a semicolon at the end of the query would cause an error.
+
+- `CDAP-4692 <https://issues.cask.co/browse/CDAP-4692>`__ - The CDAP UI now prevents users
+  from accidentally losing their DAG by showing a browser-native popup for a confirmation
+  before navigating away from the Cask Hydrator Studio view.
+
+- `CDAP-4695 <https://issues.cask.co/browse/CDAP-4695>`__ - Fixed an issue in the Windows
+  CDAP SDK where streams could not be deleted.
+
+- `CDAP-4735 <https://issues.cask.co/browse/CDAP-4735>`__ - Fixed an issue that made Java
+  extensions unavailable to programs, fixing the JavaScript-based Hydrator transforms under Java 8.
+
+- `CDAP-4908 <https://issues.cask.co/browse/CDAP-4908>`__ - Removed ``tableName`` as a
+  required setting from database sources, since the ``importQuery`` is sufficient.
+
+- `CDAP-4921 <https://issues.cask.co/browse/CDAP-4921>`__ - Renamed the Hydrator
+  ``Teradata`` batch source to ``Database``. The previous ``Database`` source is no longer
+  supported.
+
+- `CDAP-4982 <https://issues.cask.co/browse/CDAP-4982>`__ - Changed the Cask Hydrator
+  LogParser transform ``logFormat`` field from a textbox to a dropdown.
+
+- `CDAP-5041 <https://issues.cask.co/browse/CDAP-5041>`__ - Changed several
+  ``ExploreConnection`` methods to be no-ops instead of throwing exceptions.
+
+- `CDAP-5062 <https://issues.cask.co/browse/CDAP-5062>`__ - Added a ``fetch.size``
+  connection setting to the JDBC driver to control the number of rows fetched per database
+  cursor, and increased the default fetch from 50 to 1000.
+
+- `CDAP-5092 <https://issues.cask.co/browse/CDAP-5092>`__ - Fixed a problem that prevented
+  applications written in Scala from being deployed.
+
+- `CDAP-5103 <https://issues.cask.co/browse/CDAP-5103>`__ - Fix a problem so that when the
+  schema for a view was not explicitly specified, the view system metadata will include the
+  default schema for the specified format if that is available.
+
+- `CDAP-5131 <https://issues.cask.co/browse/CDAP-5131>`__ - Fixed a problem when filtering
+  plugins by their extension plugin type; filtering by the extensions plugin type was
+  returning extra results for any plugins that did not have an extension.
+
+- `CDAP-5177 <https://issues.cask.co/browse/CDAP-5177>`__ - Fixed a problem with
+  PartitionConsumer not appropriately handling partitions that had been deleted since they
+  were added to the working set.
+
+- `CDAP-5182 <https://issues.cask.co/browse/CDAP-5182>`__ - Fixed a problem preventing
```

Comment:
this release note does not match the release notes written in the JIRA:
https://issues.cask.co/browse/CDAP-5182